src/ignitus-Dashboard/Styles/style.scss

```diff
@@ -18,10 +18,10 @@
   font-size: 3rem;
   font-style: normal;
   font-weight: 500;
+  line-height: 56px;
```

Comment:
Don't use `px` for values. Use `rem` instead.

Reply:
Ok